### ISSUE TITLE
[bug] Member badge reveal showing more than 100%

### ISSUE THREAD

**Issue by norestraint** [exercism/exercism#6179]
<img width="739" alt="Screenshot 2022-01-01 at 12 10 59" src="https://user-images.githubusercontent.com/87676812/147853945-accbc49c-5b5b-494e-b2eb-ca8621245988.png">

**Comment by tejasbubane** [comment]
Looks like this is because of the hard-coded number of users: https://github.com/exercism/website/blob/9ac2ded06ed15e6dffe60bf7be4ad8a861584e9a/app/models/badge.rb#L61-L64

**Comment by iHiD** [comment]
Yeah, I'm not sure what the solution for this one is right now. `User.count` is slow enough that I don't want to use it in production. So it probably needs precaching somewhere.

**Comment by SleeplessByte** [comment]
If approximation is good enough, this is mega fast: https://dev.mysql.com/doc/refman/5.7/en/show-table-status.html

**Comment by iHiD** [comment]
This specific badge is hotfixed by https://github.com/exercism/website/pull/2330

I'll look at @SleeplessByte's suggestion for the wider thing.
